packages/core/echo/echo-typegen/src/codegen.ts

```diff
@@ -125,20 +125,10 @@ export const generate = (root: pb.NamespaceBase): string => {
   export const schema = ${importNamespace}.EchoSchema.fromJson(schemaJson);
 
   ${declarations}
-
   `;
 };
 
 function* emitDeclarations(ns: pb.ReflectionObject): Generator<string> {
-  if ((ns instanceof pb.Namespace || ns instanceof pb.Type) && ns.nestedArray.length > 0) {
-    yield text`
-      export namespace ${ns.name} {
-        ${ns.nestedArray.flatMap((nested) => Array.from(emitDeclarations(nested)))}
-      }
-      
-    `;
-  }
-
   if (ns instanceof pb.Type) {
     if (ns.name === 'Text') {
```

Comment:
extract any string literals that are used more than once?

Reply:
Sure